sdks/python/apache_beam/yaml/README.md

```diff
@@ -527,3 +527,23 @@ in a yaml file as
 ```
 python -m apache_beam.yaml.main --yaml_pipeline_file=/path/to/pipeline.yaml [other pipeline options such as the runner]
 ```
+
+## Logging
+
+The logging level can be set to increase or decrease logging output during
+pipeline translation.
+
+```
+pipeline:
+  transforms:
+    - type: ReadFromCsv
+      config:
+        path: /path/to/input*.csv
+    - type: WriteToJson
+      config:
+        path: /path/to/output.json
+      input: ReadFromCsv
+logging: ERROR
```

Comment:
I'm not sure it makes sense to have this as a top-level attribute in the yaml. (E.g. it doesn't impact pipeline execution, just the construction). 

We could make it an arg for the template (likely defaulting to INFO as the only reason one would look at the template logs is if something went wrong), and normal invocation of the main.py should probably not set the logging high (but one could set it via an arg or programmatically).